### PR TITLE
Configure daemon CD

### LINKED ISSUE #159: Configure daemon CD

### PR DESCRIPTION
**Pull request by paulobressan**
Close #159

### CODE REVIEW

File: .github/iac/main.tf [hunk 1]
```diff
@@ -76,3 +82,17 @@ module "fabric_rpc" {
   email_ses_verified_email    = local.email_ses_verified_email
   url_prefix                  = "rpc"
 }
+
+module "fabric_daemon" {
+  source = "../../bootstrap/daemon"
+
+  namespace      = local.daemon_namespace
+  image          = var.daemon_image
+  cluster_id     = ""
```
Comment (gonzalezzfelipe) [inline .github/iac/main.tf:91]: This should be an identifier for the m2 cluster. I would put `txpipe-us-east-2-m2`

File: .github/iac/main.tf [hunk 2]
```diff
@@ -34,13 +35,18 @@ variable "slack_webhook_url" {}
...
-  namespace                   = "demeter-global"
+  rpc_namespace               = "demeter-global"
+  daemon_namespace            = "demeter-system"
   replicas                    = 1
   broker_urls                 = "cs01k9hsakr3s8si3o00.any.us-east-1.mpx.prd.cloud.redpanda.com:9092"
   secret                      = var.secret
   kafka_rpc_username          = "rpc"
   kafka_rpc_password          = var.kafka_rpc_password
+  kafka_daemon_username       = "daemon"
+  kafka_daemon_password       = var.kafka_daemon_password
```
Comment (gonzalezzfelipe) [inline .github/iac/main.tf:49]: I'll pass you this values on slack

Comment (paulobressan) [reply .github/iac/main.tf:49]: tks

File: .github/iac/main.tf (hunk 1 above)
Comment (paulobressan) [reply .github/iac/main.tf:91]: cool! I will update it

**Pull request merged**